app/share/views.py

```diff
@@ -88,6 +96,332 @@ def resources():
 @share.route('/files/<model_run_uuid>')
 @login_required
 def files(model_run_uuid):
+
+    if not os.path.exists(app.config['UPLOAD_FOLDER']):
+        os.makedirs(app.config['UPLOAD_FOLDER'])
+
     "View of file submission for as yet unselected resource to add to"
     model_run_uuid = model_run_uuid
     return render_template('share/files.html', model_run_uuid=model_run_uuid)
+
```

Comment:
What I understood is the page should include only the browse option. When the user selects 'Add files to this resource option', redirect to a page that has browse option to upload 2 files. When the user selects 'Upload', insert input files to the VW. Let me know if its right. Or, as you mentioned before, I can add a dropdown list which has names of 2 models. When the user selects 'PRMS', go to the /share/files/<uuid> page which already exists.